src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -94,6 +94,13 @@
      */
     @Getter
     public boolean backpointersDisabled = false;
+    /**
+     * To prevent livelock condition in fetchLayout.
+     * Notifies that the runtime is no longer used
+     * and retries to fetch the layout can be stopped.
```

Comment:
I don't think this is necessary. It's also not a livelock condition.
The runtime calls an rpc on a single server every time it needs to fetch a layout.
The layout fetch is spun in a common pool, initiating a proper shutdown of that pool is more appropriate
than signaling using global variables.